packages/engine/src/worker/runner/comms/mod.rs

```diff
@@ -99,6 +111,10 @@ impl TargetedRunnerTaskMsg {
         let target = task_msg.target().into();
         let package_id = (task_msg.package_sid() as usize).into();
 
+        log::trace!(
+            "Outbound task payload string: {:?}",
```

Comment:
```suggestion
            "Outbound task payload: {:?}",
```

Reply:
I would leave this, because it distinguishes the payload as a string, before deserialization to JSON, from the deserialized payload (in case errors occur during deserialization).